tests/__setup__/resolver.ts

```diff
@@ -0,0 +1,31 @@
+// This code is from
```

Comment:
is it possible to use moduleNameMapper instead like suggested here? https://github.com/uuidjs/uuid/pull/616#issuecomment-1114696186
It might be a little easier to find and remove later on

Reply:
It's ridiculous they haven't closed out https://github.com/uuidjs/uuid/pull/616 for 4 months

Reply:
it is possible to use `moduleNameMapper`, but it seemed like that might have some consequences, discussed here: https://github.com/uuidjs/uuid/pull/616#issuecomment-1111012599